recipes/ncrystal/build.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -x
+set -e
+which cmake
+cmake --version
+$PYTHON --version
```

Comment:
These are well defined in conda-forge and logged. I'm saying you have to remove but you probably don't need them here.